src/util.cpp

```diff
@@ -478,7 +478,7 @@ static CCriticalSection csPathCached;
 
 static boost::filesystem::path ZC_GetBaseParamsDir()
 {
-    // Copied from GetDefaultDataDir and adapter for zcash params.
+    // Copied from GetDefaultDataDir and adapter for Zcash params.
```

Comment:
Making a change like this can/will cause merge conflicts every time we merge upstream Zcash changes into Hush. Please avoid stuff like this.